app/javascript/react/screens/App/Overview/components/Migrations/MigrationsInProgressCard.js

```diff
@@ -19,6 +19,60 @@ const MigrationsInProgressCard = ({ plan, allRequestsWithTasks, reloadCard, hand
   const requestsOfAssociatedPlan = allRequestsWithTasks.filter(request => request.source_id === plan.id);
   const mostRecentRequest = requestsOfAssociatedPlan.length > 0 && getMostRecentRequest(requestsOfAssociatedPlan);
 
+
+  // === TODO FIXME-- THE BELOW IS MOCK MUTATION CODE TO BE REMOVED ===
+  // This code:
+  // * sticks a fake active pre-playbook in the first migration on the screen
+  // * sticks a fake active post-playbook in the third migration on the screen
+  const a = window.dangerousGlobalAccumulator;
+  const mockTasks = mostRecentRequest.miq_request_tasks.map((task, i) => ({
+    ...task,
+    options: {
+      ...task.options,
+      playbooks: {
+        pre: {
+          job_id: 4,
+          status: (a === 0 && i === 0 ? "Active" : "Succeeded"),
+          last_task: "Task name"
+        },
+        post: {
+          job_id: 5,
+          status: (a === 2 && i === 0 ? "Active" : "Succeeded"),
+          last_task: "Task name"
+        }
+      }
+    }
+  }));
+  window.dangerousGlobalAccumulator++;
+  const mostRecentTasks = mockTasks;
+  // ^^^ TODO FIXME-- THE ABOVE IS MOCK MUTATION CODE TO BE REMOVED ^^^
+  // We should remove the above code when the real API data is in place.
+  
+  // const mostRecentTasks = mostRecentRequest.miq_request_tasks;
+
+  const getActivePlaybook = task => {
+    if (!task || !task.options || !task.options.playbooks) return {};
+    const { options: { playbooks } } = task;
+    if (playbooks.pre.status === 'Active') return playbooks.pre;
```

Comment:
it would be nice if we can create shared constants for pre/post migration status checks... you will probably want to sync up w/ @michaelkro when he gets back from PTO on this.

Reply:
Yeah that's my thinking too. I'll add a comment to that effect.